code/modules/antagonists/space_ninja/space_ninja.dm

```diff
@@ -118,7 +118,6 @@
 	operative.dna.update_dna_identity()
 	operative.dna.species.pre_equip_species_outfit(null, operative)
 	operative.regenerate_icons()
-	SSquirks.AssignQuirks(operative, operative.client, TRUE, TRUE, null, FALSE, operative)
```

Comment:
Bubber edit this

Reply:
![image](https://github.com/user-attachments/assets/f7f5a24a-8fcf-44b6-9740-23cfa5f6deed)